packages/dx-react-chart/docs/guides/zoom-and-pan.md

```diff
@@ -0,0 +1,31 @@
+# React Chart - Zoom and Pan
+
+Users can zoom and pan (scroll) the React Chart. These actions change the chart's viewport. Mouse and touch gestures are supported.
+
+## Basic Setup
+
+Import the [ZoomAndPan](../reference/zoom-and-pan.md) plugin and add it to the `Chart` component to enable zooming and panning.
+
+Note that the [plugin order](./plugin-overview.md#plugin-order) is important.
+
+### Uncontrolled Mode
+
+In uncontrolled mode, add the `ZoomAndPan` plugin to the `Chart` component.
+
+.embedded-demo({ "path": "zoom-pan/uncontrolled", "showThemeSelector": true })
+
+### Controlled Mode
+
+In controlled mode, pass the viewport's boundaries to the `ZoomAndPan` plugin's `viewport` property and handle `onViewportChange` event to control zoom and pan externally.
+
+.embedded-demo({ "path": "zoom-pan/controlled", "showThemeSelector": true })
+
+## Enable Zooming or Panning Inidividually
+
+Use the `interactionWithArguments` and `interactionWithValues` properties to enable zooming or panning individually for corresponding scales.
```

Comment:
"zooming or panning individually" Explain.

Reply:
You can enable either zooming or panning or both for individual scales.

Reply:
You can enable zooming or panning or both for individual scales.
```suggestion
Use the `interactionWithArguments` and `interactionWithValues` properties to enable zooming or panning or both for individual scales.
```